docs/index.rst

```diff
@@ -250,7 +247,7 @@ Status
 +----------------------+----------------+-------------+
 | License              | |MIT|          |             |
 +----------------------+----------------+-------------+
-| Community            | |slack|        | |gitter|    |
+| Community            | |slack|        |             |
```

Comment:
Perhaps we can add a link to Discussions in place of gitter here?

Reply:
Agreed.  We should add this.  @tkoyama010, would you mind adding in the link here?  Would be nice to have a full table as well.